### PR TITLE
Add FAQs

### PR DESCRIPTION
**Pull request by edoardob90**
Adds FAQs for the upcoming tutorial. Might change/be expanded in the future

### CODE REVIEW

**Comment by edoardob90** [comment]
A quick comment about the choice of Slack. I figured out that it might be better than Zulip for two reasons:

- I received a few questions like "Where the discussion platform will be hosted? Are we going to use Slack or Teams?"
- There was already a workspace for the tutorial, and I realized that many people already have a Slack account with the `@empa.ch` email address.

I think it's simply the easiest option.

**Review by baffelli — DISMISSED** [review]
Everything ok from my side

File: FAQ.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# FAQs for Python Training Course
+
+## What will be the workshop schedule?
+
+Each tutorial session lasts 4 half-days, from 9:00 to 13:00 (breaks included). During one afternoon, a one hour and a half will be dedicated to the extra session on Linux & Gitlab.
+
+## Which operating system should I use?
+
+There is no specific requirement for the operating system. The training will be conducted using Jupyter notebooks, which are browser-based, so an updated browser is the only requirement. Chrome-based or Firefox usually work better with Jupyter notebooks, but this is only a suggestion.
```
Comment (yakutovicha) [inline FAQ.md:9]:
```suggestion
There is no specific requirement for the operating system. The training will be conducted on [binder](https://mybinder.org/) using Jupyter notebooks, which are browser-based, so an updated browser is the only requirement. Chrome-based or Firefox usually work better with Jupyter notebooks, but this is only a suggestion.
```

**Review by yakutovicha — COMMENTED** [review]
all good. Just a minor request.

**Review by yakutovicha — APPROVED** [review]

**Pull request merged**